eif_utils/src/bin/eif_build.rs

```diff
@@ -122,6 +141,27 @@ fn main() {
         _ => panic!("Both signing-certificate and private-key parameters must be provided"),
     };
 
+    let img_name = matches.value_of("image_name").map(|val| val.to_string());
+    let img_version = matches.value_of("image_name").map(|val| val.to_string());
+    let metadata = matches.value_of("metadata").map(|val| val.to_string());
+
+    let arg_data = EifIdentityInfo {
+        img_name: match img_name {
+            Some(name) => name,
+            // Set default value kernel file name
+            None => {
+                let path_split: Vec<&str> = kernel_path.split('/').collect();
+                path_split[path_split.len() - 1].to_string()
```

Comment:
What does this do? If it tries to get the last bit, use `Path` and get the `file_name`.

Reply:
fixed